eval/eval_test.go

```diff
@@ -20,7 +20,10 @@ func TestEvalIntegerExpression(t *testing.T) {
 		{"10", 10},
 		{"-5", -5},
 		{"-10", -10},
-		{"5 + 5 + 5 + 5 - 10", 10},
+		{"5 + 5 + 5 + 5 - 10 /* some block comment */", 10},
+		/* These don't work, we need to make comment a identity operator or prune them entirely from the AST. */
+		// {"5 + /* block comment in middle of expression */ 2", 7},
+		// {" - /* inline of prefix */ 5", -5},
```

Comment:
@ccoVeille also this ^ ie the block/inline comments currently do break expressions, basically act as a ; or \n and making them not do that requires more surgery (like 2 passes, one for -format/pretty print and one for nuking the comments... which seems not ideal)